_config.yml

```diff
@@ -1,111 +1,78 @@
-# Website title - used in the <title> tag of the <head> section of the website.
 title: Linaro
-# Description of the website used in the Meta Description.
-description: > # this means to ignore newlines until "baseurl:"
+url: https://www.linaro.org
+baseurl: ""
+description: |-
   Linaro is a collaborative engineering organization consolidating and optimizing open source software and tools for the Arm architecture.
-# Destination of the Jekyll site upon build of site.
 destination: _site
-# Include the Linaro Jekyll Theme
-theme: jumbo-jekyll-theme
-# Permalink for the blog posts of the Jekyll website.
 permalink: /:categories/:title/
-# Jekyll Responsive Images dir
-blog_images_dir: assets/images/blog/
-email: contact@linaro.org
-# Syntax highlighter for 96boards blog
+theme: jumbo-jekyll-theme
 highlighter: rouge
-# Markdown version
 markdown: kramdown
-# production status for robots.txt and files alike.
-production: false
 kramdown:
   input: GFM
-  syntax_highlighter: rouge # Syntax highlighter for Kramdown.
-# Encoding of the files being included in the Jekyll Build.
+  syntax_highlighter: rouge
 encoding: utf-8
-# Jekyll configuration for running the site in "Safe" mode.
 safe: false
-# Search label setting for site search with e.g. search/?fields.label=96Boards
-search-label: Linaro
-# Disqus Comments settings.
-disqus: true
-disqus_shortname: "Linaro"
-# This is the url that is used when your site is built for canonicals etc.
-url: "https://www.linaro.org"
-# url: "https://www.site-test.linaro.org"
-
-# Make sure that Liquid errors stop the build
-liquid:
-  error_mode: strict
-
-# Shema.org Setup -> used in the schema.html include.
-schema:
-  organisation:
-    name: Linaro
-    url: https://www.linaro.org
-    sameAs:
-      - https://twitter.com/linaroorg
-      - https://www.facebook.com/LinaroOrg
-      - https://www.linkedin.com/company/1026961/
-      - https://plus.google.com/+LinaroOnAir
-      - https://github.com/Linaro
-      - https://www.slideshare.net/linaroorg
-    address:
-      street: Harston Mill, Royston Rd
-      town: Harston
-      county: Cambridge
-      post_code: CB22 7GG
-      country_code: UK
-# Files that should be excluded from the Jekyll build.
 exclude:
-  - _site
+  - Gemfile
+  - Gemfile.lock
+  - .asset_pipeline
   - build-site.sh
   - check-links.sh
   - CODEOWNERS
-  - Gemfile
-  - Gemfile.lock
-  - README.md
```

Comment:
This one needs to be added back,

Reply:
Those have been added back in now. Just waiting on these statuses to resolve.